lambda/create-case/problem.go

```diff
@@ -31,6 +41,20 @@ var ProblemInvalidRequest Problem = Problem{
 }
 
 func (problem Problem) Respond() (events.APIGatewayProxyResponse, error) {
+	err := json.NewEncoder(os.Stdout).Encode(LogEvent{
+		ServiceName: "opg-data-lpa-uid",
+		Timestamp:   time.Now(),
+		Status:      problem.StatusCode,
+		Problem:     problem,
+	})
+
+	if err != nil {
+		return events.APIGatewayProxyResponse{
+			StatusCode: 500,
+			Body:       "{\"code\":\"INTERNAL_SERVER_ERROR\",\"detail\":\"Internal server error\"}",
+		}, nil
+	}
```

Comment:
I think we can probably ignore this `err`: if the logging fails, it isn't critical and it would be better to return the original error (e.g. the 400) to the user rather than a generic Internal server error.